package-template/.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish Package for the First Time
+
+on:
+  workflow_dispatch:
+
+env:
+  PPR_AUTH: ${{ secrets.PPR_AUTH }}
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+    # We need to have at least one tag available prior to publication. So we will publish the first release
+    # during publication.
+    - name: Checkout the Latest Package Code
+      uses: actions/checkout@v3
+    - name: Get Version
+      id: version
+      run: node -p "require('./package.json').version" >> "$GITHUB_OUTPUT"
```

Comment:
things printed to `$GITHUB_OUTPUT` are a key value pair as used to specify env vars, so like `version=1.0.0`, and then retrieved by the key, like `steps.version.outputs.version`

Reply:
Thanks a ton! Somehow missed this